codespell_lib/data/dictionary.txt

```diff
@@ -860,7 +861,7 @@ afficionados->aficionados
 affilate->affiliate
 affilates->affiliates
 affilliate->affiliate
-affinitied->affinitized
+affinitied->affinities, affinitized,
```

Comment:
Got a hit for the -zed thing? Does -sed exist too?

Reply:
I'm not sure. I don't really get examples for affinitized.

Reply:
Then surely in the interest of fixing mistakes we should remove it?

Reply:
removed